.changeset/dull-birds-lie.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend': minor
+---
+
+**BREAKING** The non-PR/MR Git Actions now return the commitHash of the commit pushed as a new output called `commitHash`
```

Comment:
```suggestion
**BREAKING** The non-PR/MR Git Actions now return the commit hash of the commit pushed as a new output called `commitHash`
```

Avoid camelCase for plain english, code blocks for code 👍

Reply:
I'm not sure that this is actually a breaking change? You can still keep it as minor as it's `1.x` but, remove the `breaking` from the changeset too.

Reply:
I thought this might be breaking as the API changes, but as most consumers are internal we could also skip this, if you would like me to